docs/source/concepts/builder.rst

```diff
@@ -9,6 +9,26 @@ The Builder
    :local:
    :backlinks: none
 
+Users of the Vivarium framework build simulations with components. Components are
+Python classes that represent aspects and behaviors of simulants. Components can be
+leveraged directly from modules like Vivarium Public Health or be user-written
```

Comment:
Link to vph main doc page and drop subclass.  `Vivarium Public Health <https://...>`_